src/DGmethods/DGmodel.jl

```diff
@@ -8,124 +8,110 @@ end
 
 function (dg::DGModel)(dQdt, Q, param, t; increment=false)
   bl = dg.balancelaw
+  DFloat = eltype(Q)
   device = typeof(Q.Q) <: Array ? CPU() : CUDA()
 
   grid = dg.grid
-  topology = grid.topology
-
-  dim = dimensionality(grid)
-  N = polynomialorder(grid)
-  Nq = N + 1
-  Nqk = dim == 2 ? 1 : Nq
-  Nfp = Nq * Nqk
-  nrealelem = length(topology.realelems)
-
-  Qvisc = param.diff
-  auxstate = param.aux
+  Ω    = grid.topology.realelems
+  nΩ   = length(Ω)
+  Nᵈ   = dimensionality(grid)
```

Comment:
`Nd` or just `d` as this might be easier to type regularly (this matches `Nq`, `Np`, etc. below)

Reply:
yeah for this variable I can see that. I generally prefer using superscript where you would do so in writing an equation, but there are so many of these N variables that it is hard to do it properly

Reply:
switched to using `Nd`